src/components/SearchResultItem.tsx

```diff
@@ -22,12 +23,23 @@ export class SearchResultItem extends React.Component<SearchResultItemProps, Sea
         let highLightedItemText = getHighlightedText(this.props.data.text, this.props.highlightedText, true);
         let highLightedCategoryText = getHighlightedText(this.props.category, this.props.highlightedText, false);
         let ItemTypeIconPath = "src/resources/icons/library-" + this.props.data.itemType + ".svg";
+        let ItemDescription: JSX.Element = null;
+
+        if (this.props.detailed) {
+            let description = "No description available";
```

Comment:
Currently, dynamo shows "no description available" if there is no description. Should we follow the current dynamo or just don't show description when there is no description?
![capture](https://cloud.githubusercontent.com/assets/14089267/26040736/61f2ce60-395a-11e7-8ad2-a193921ea7fe.PNG)